hedera-node/src/main/java/com/hedera/services/context/properties/GlobalDynamicProperties.java

```diff
@@ -20,10 +20,18 @@
  * ‍
  */
 
-public class NoFeeScheduleExistsException extends Exception {
-	private static final long serialVersionUID = 3889224951972943501L;
+public class GlobalDynamicProperties {
+	private final PropertySource properties;
 
-	public NoFeeScheduleExistsException(String message) {
-		super(message);
+	public GlobalDynamicProperties(PropertySource properties) {
+		this.properties = properties;
+	}
+
+	public int maxTokensPerAccount() {
+		return 1_000;
+	}
+
+	public int maxTokenSymbolLength() {
+		return 10;
```

Comment:
It would be better if these  values can be configurable later.

Reply:
I'm working on that now! 👍